services/management/service.go

```diff
@@ -42,14 +42,16 @@ var serviceTypes = map[inventorypb.ServiceType]models.ServiceType{
 type ServiceService struct {
 	db       *reform.DB
 	registry agentsRegistry
+	state    agentsStateUpdater
 	vmdb     prometheusService
 }
 
 // NewServiceService creates ServiceService instance.
-func NewServiceService(db *reform.DB, registry agentsRegistry, vmdb prometheusService) *ServiceService {
+func NewServiceService(db *reform.DB, registry agentsRegistry, state agentsStateUpdater, vmdb prometheusService) *ServiceService {
 	return &ServiceService{
 		db:       db,
 		registry: registry,
```

Comment:
we don't use registry here anymore, please remove `registry` in all places where we don't use it.